addons/hr_expense/models/hr_expense.py

```diff
@@ -169,12 +193,13 @@ def _compute_amount_residual(self):
 
     @api.depends('date', 'total_amount', 'currency_id', 'company_currency_id')
     def _compute_total_amount_company(self):
+        date_today = fields.Date.today()
```

Comment:
Why you don't use context_today like `_compute_label_convert_rate`  ?

Reply:
Thanks for pointing it out :) changed!